code/modules/research/designs/weapon_designs.dm

```diff
@@ -2,6 +2,26 @@
 /////////////////Weapons/////////////////
 /////////////////////////////////////////
 
+/datum/design/nuclear_gun/pin_testing
```

Comment:
what's with these funky paths? why is a pin a subtype of the nuclear_gun design datum?